tracer/test/snapshots/Iast.SSRF.AspNetCore2.IastEnabled.verified.txt

```diff
@@ -0,0 +1,81 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: GET /iast/ssrf,
+    Service: Samples.Security.AspNetCore2,
+    Type: web,
+    Tags: {
+      aspnet_core.route: iast/ssrf,
+      component: aspnet_core,
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.route: iast/ssrf,
+      http.status_code: 200,
+      http.url: http://localhost:00000/Iast/SSRF?host=localhost,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      _dd.iast.enabled: 1,
+      _dd.iast.json:
+{
+  "vulnerabilities": [
+    {
+      "type": "SSRF",
+      
+      "evidence": {
+        "valueParts": [
+          {
+            "value": "https://user:password@"
```

Comment:
Should the password be redacted here?

Reply:
In this test, redaction is disabled. The redacted version can be found at Iast.SSRF.AspNetCore2.IastEnabled.RedactionEnabled.verified.txt